react/components/Collapsible/index.js

```diff
@@ -5,6 +5,17 @@ import CaretDown from '../icon/CaretDown'
 import CaretUp from '../icon/CaretUp'
 import { jsFocusVisible } from './styles.css'
 
+const colorMap = {
+  success: 'c-success',
+  danger: 'c-danger',
+  warning: 'c-warning',
+  black: 'black',
```

Comment:
I think both black and white tokens are deprecated

Reply:
also make sure each of these colors make sense on a caret. If a designer has approved this then ok, just be sure if there is a use case for, say, "warning"